zebra/zebra_pbr.c

```diff
@@ -156,118 +156,43 @@ void zebra_pbr_rules_free(void *arg)
 
 uint32_t zebra_pbr_rules_hash_key(const void *arg)
 {
-	const struct zebra_pbr_rule *rule;
+	const struct zebra_pbr_rule *rule = arg;
 	uint32_t key;
 
-	rule = arg;
-	key = jhash_3words(rule->rule.seq, rule->rule.priority,
-			   rule->rule.action.table,
-			   prefix_hash_key(&rule->rule.filter.src_ip));
-
-	key = jhash_3words(rule->rule.filter.fwmark, rule->vrf_id,
-			   rule->rule.filter.ip_proto, key);
+	key = jhash_2words(rule->rule.unique, rule->sock, 0);
```

Comment:
the tuple of vrf+ifname is necessary, I think? small change, but will be safer.

Reply:
I added vrf_id back to the hash in this latest push